portal/pom.xml

```diff
@@ -131,7 +131,8 @@
             <resource>
               <directory>${project.parent.basedir}/business/src/main/resources</directory>
               <targetPath>WEB-INF/classes</targetPath>
-              <filtering>true</filtering>
+              <!-- do not hard-set portal.properties config at build-time -->
+              <filtering>false</filtering>
```

Comment:
this might be a thing for pipelines to check if this is fine

Reply:
I guess we will check this during the next importer build. We definitely use filtering, but I would guess that the filtering we need is done inside the (spring-based) importer codebase and that the importer pom.xml is the controlling file.